Source/WebKit/UIProcess/Launcher/haiku/ProcessLauncherHaiku.cpp

```diff
@@ -52,26 +52,30 @@ status_t processRef(BString path, entry_ref* pathRef)
 
 void ProcessLauncher::launchProcess()
 {
-    BString executablePath;
+    BString executablePath,executableSignature;
 
     switch (m_launchOptions.processType) {
     case ProcessLauncher::ProcessType::Web:
         executablePath = executablePathOfWebProcess();
+        executableSignature = "application/x-vnd.haiku-webkit.webprocess";
```

Comment:
what is this used for?

Reply:
I have created a common bapplication if you see in auxiliaryprocessmainhaiku.h you can see process app so to initialize them we would need signatures